src/ui/navbar-header.svelte

```diff
@@ -9,28 +11,57 @@
   export let shouldShowDetailsButton = false;
   export let expanded = false;
   export let toggle;
+  export let showCloseButton: boolean;
+  export let onClose: (() => void) | undefined = undefined;
 </script>
 
 <ModalSection as="header">
-  <div class="rcb-header-layout">
-    {@render children?.()}
-    {#if shouldShowDetailsButton}
-      <button
-        type="button"
-        class="rcb-header-details"
-        on:click={toggle}
-        on:keydown={(e) => (e.key === "Enter" || e.key === " ") && toggle()}
-        aria-expanded={expanded}
-        aria-controls="rcb-header-details-content"
-      >
-        <Localized key={LocalizationKeys.NavbarHeaderDetails} />
-        <IconArrow className={expanded ? "expanded" : "collapsed"} />
-      </button>
+  <div class="rcb-header-multiline-layout">
+    {#if showCloseButton}
+      <div class="rcb-back">
+        <BackButton
```

Comment:
should we not show the backbutton is `onClose` is not defined? (although it will always be defined I guess)

Reply:
I think the current usage is safe.
I usually prefer to not assume on the existence of a parameter but rather explicitly mark as show/not show.